bin/infection

```diff
@@ -38,30 +38,30 @@ use Infection\Console\Application;
 use Infection\Container;
 use function Safe\getcwd;
 
-if (in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) === false) {
-    echo PHP_EOL . 'Infection may only be invoked from a command line, got "' . PHP_SAPI . '"' . PHP_EOL;
+if (\in_array(\PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) === false) {
```

Comment:
All of those could be use statements instead as well

Reply:
CS Fixer plainly refuses to make them so.

Reply:
I confirm - when I use import here and run `make cs`, it removes import and adds `\` 

```
1)) /home/maksrafalko/apps/infection/bin/infection (native_function_invocation, no_unused_imports)
```

On the other side, it works correctly for for example `SourceDirGuesser.php` - when I remove import there, fixer adds it back and doesn't add `\`:

```
1) /home/maksrafalko/apps/infection/src/Config/Guesser/SourceDirGuesser.php (native_function_invocation, global_namespace_import, ordered_imports)
```

it looks like a big in PHP-CS-Fixer